examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,12 @@
+''' An example demonstration the use of `hspan`, `wspan`, and `vstrip`.
+It also uses the `HoverTool`.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.HoverTool
+    :refs: :ref:`ug_basic_hspan`
```

Comment:
This reference does no exist, as far as I know. I don't think think there are currently direct refs for these, so maybe jsut delete this line entirely. cc @tcmetzger for any other suggestions